packages/medusa/src/migrations/1699371074198-drop-non-null-constraint-price-list.ts

```diff
@@ -0,0 +1,19 @@
+import { MigrationInterface, QueryRunner } from "typeorm"
+
+export class DropNonNullConstraintPriceList1699371074198
+  implements MigrationInterface
+{
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`
+      ALTER TABLE IF EXISTS price_list
+      ALTER COLUMN name DROP NOT NULL;
+    `)
+  }
+
+  public async down(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`
+      ALTER TABLE IF EXISTS price_list
+      ALTER COLUMN name DROP NOT NULL;
```

Comment:
question: should be the inverse of the upper query i.e. should we add not null constraint?